src/Communities.re

```diff
@@ -0,0 +1,100 @@
+module Query = [%relay.query
+  {|
+    query CommunitiesQuery {
+      # communities_connection(first: 1000, where: { name: { _like: "%test%" } }) -- adding a filter breaks local store updater
```

Comment:
Is there a solution to this issue?

Reply:
The solution is either adding an empty "filters"-list to the @connection directive in case one is sure that this list should always be updated after the mutation, or adding "where" as a filter in the list and specifying the exact same filter in the connections passed to `createAndAddEdgeToConnections`.
More info here: https://relay.dev/docs/en/experimental/a-guided-tour-of-relay#connection-identity-with-filters